Lib/bdb.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, skip=None):
         self.skip = set(skip) if skip else None
         self.breaks = {}
         self.fncache = {}
+        self._curframe = None
+        self.lasti = -1
```

Comment:
```suggestion
        self.f_lasti = -1
```
Might be more clear that it corresponds to `frame.f_lasti`

Reply:
The reason I went for `lasti` instead of `f_lasti` was that's how [`dis.disassemble`](https://docs.python.org/3/library/dis.html#dis.disassemble) uses. Also in `pdb`, [`lasti2lineno`](https://github.com/python/cpython/blob/b4978ff872be5102117b4e25d93dbbb4e04c8292/Lib/pdb.py#L110) uses `lasti`. The `f` in `f_lasti` is a prefix for frame attributes, and I think `lasti` is a reasonable choice here.